fiware_cloto/cloto/manager/InfoManager.py

```diff
@@ -26,6 +26,8 @@
 from django.core.exceptions import ValidationError
 from django.conf import settings
 
+from pika import exceptions
```

Comment:
Imports were made following "Official guide" from rabbitMQ. I do not check if this import is needed.